ZeldaFullEditor/ZSCustomOverworld.asm

```diff
@@ -113,6 +113,7 @@ Palette_ArmorAndGloves                     = $1BEDF9
 Palette_Hud                                = $1BEE52
 Palette_OverworldBgAux3                    = $1BEEA8
 Palette_OverworldBgMain                    = $1BEEC7
+PaletteData_owmain                         = $1BE6C8
```

Comment:
nit: PaletteData_OwMain

Reply:
Thats to match the disassembly name. I agree I'll change that later though.